.github/workflows/build.yml

```diff
@@ -7,19 +7,23 @@ on:
       - '**'
     paths-ignore:
       - 'CHANGES'
-      - 'openvdb/doc/**'
+      - 'doc/**'
       - 'openvdb_maya/**'
+      - 'openvdb_houdini/**'
       - 'openvdb_ax/**'
-      - '**/*.md'
+      - '**.md'
+      - '**.txt'
```

Comment:
Won't this miss changes to CMakeLists.txt?

Reply:
Yes, will fix this